src/client.ts

```diff
@@ -497,6 +499,7 @@ export function createClient(options: ClientOptions): Client {
             // either the canceller will be called and the promise resolved
             // or the socket closed and the promise rejected
             await throwOnCloseOrWaitForCancel(() => {
+              cancellerRef.current = null;
```

Comment:
Uhm, at this point the canceller is already gone 😄 . This is what I had in mind with https://github.com/enisdenjo/graphql-ws/pull/67#pullrequestreview-529489737: https://github.com/enisdenjo/graphql-ws/pull/67/commits/fe3c6f64b834287174e2b0d45b4429c13ef458e9.

Reply:
That definitely makes more sense.  😄